addons/popochiu/engine/objects/character/popochiu_character.gd

```diff
@@ -811,14 +810,21 @@ func _get_valid_oriented_animation(animation_label):
 	return null
 
 
-func _walk_to_node(node: Node2D, offset: Vector2) -> void:
+func _walk_to_node(node: Node2D, offset: Vector2, is_blocking := false) -> void:
```

Comment:
I don't want to become overly pedantic, but boolean params like these should be avoided because they make the calls to the function unreadable. What does `true` stand for?

It may be better to have constants like `ACT_MODE_BLOCKING`, `ACT_MODE_NON_BLOCKING`, so that:

1. It's always clear what you are attempting
2. You can push those down to public methods, if the need be, because of point 1
3. Should you stumble onto another necessary mode of execution, it's easier to add a constant; with bools you're stuck with a two-way switch

Reply:
I get your point. Right now I can not imagine another mode of execution appart from BLOCKING and NON-BLOCKING, but I agree that a new one could appear in the future. I'm gonna make the change because I like the idea of making things more readable.